app/WebParsing/UtsgJsonParser.hs

```diff
@@ -0,0 +1,251 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+{-
+module WebParsing.UtsgJsonParser
+     (getJSON) where
+-}
+
+import Data.Aeson
```

Comment:
Please only import the functions/types you need (other than for `Database.Tables`).